modules/core/04-channel/v2/keeper/packet.go

```diff
@@ -288,8 +288,8 @@ func (k *Keeper) timeoutPacket(
 	// that the packet was indeed sent by our counterparty.
 	channel, ok := k.GetChannel(ctx, packet.SourceChannel)
 	if !ok {
-		// TODO: figure out how aliasing will work when more than one packet data is sent.
-		channel, ok = k.convertV1Channel(ctx, packet.Data[0].SourcePort, packet.SourceChannel)
+		// TODO: figure out how aliasing will work when more than one payload is sent.
+		channel, ok = k.convertV1Channel(ctx, packet.Payloads[0].SourcePort, packet.SourceChannel)
```

Comment:
not related, but aliasing fallback logic doesn't need to happen in timeout, the alias should have been set in `SendPacket`.

Reply:
https://github.com/cosmos/ibc-go/issues/7488